src/attributes.rs

```diff
@@ -118,6 +118,9 @@ pub fn from_fn_attrs<'gcc, 'tcx>(
         if codegen_fn_attrs.flags.contains(CodegenFnAttrFlags::COLD) {
             func.add_attribute(FnAttribute::Cold);
         }
+        if codegen_fn_attrs.flags.contains(CodegenFnAttrFlags::FFI_RETURNS_TWICE) {
+            to_add.push(FnAttribute::ReturnsTwice);
```

Comment:
It should be instead:
```suggestion
            func.add_attribute(FnAttribute::ReturnsTwice);
```